.github/workflows/cli-tox.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: [self-hosted, linux, X64]
     strategy:
       matrix:
-        python: ["3.8", "3.10"]
```

Comment:
I'm fine with moving it to core22, but can we keep 3.8 in the matrix for unit tests for now? I think we might have some users that still need to run from source on older systems. Fortunately not so old that 3.8 has been a problem for them though.